README.md

```diff
@@ -12,6 +12,29 @@ problem of requiring numpy and cython for setup.py to execute. The package is
 named iow in pypi as "bp" was taken at time of registration.
 
 ```
-conda install numpy cython
-pip install iow
+$ conda install numpy cython
+$ pip install iow
+```
+
+Fragment insertion
+------------------
+
+BP supports the [jplace format](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0031009). Fragments can be inserted using either fully-resolved or multifurcation mode to resolve multiple placements to the same edge. In fully resolved, the edge placed against is broken N times where N is the number of fragments on the edge. In multifurcation, a new node is constructed as the average of the distal length for the N fragments, and a separate multifurcation node is added which encompasses the placed fragments.
+
+Insertions can be handled by the command line following install:
+
+```
+$ bp placement --help
+Usage: bp placement [OPTIONS]
+
+Options:
+  --placements PATH               jplace formatted data  [required]
+  --output PATH                   Where to write the resulting newick
+                                  [required]
+
+  --method [fully-resolved|multifurcating]
```

Comment:
I'm wondering if it somewhere in the documentation the difference between the options in this parameter should be explained. To be honest, I don't know the difference.

Reply:
kk

Reply:
actually, there is a description on line 22, is that good enough?